observant-otters/forumthing/forum/models.py

```diff
@@ -1,3 +1,31 @@
 from django.db import models
+from user.models import User
 
-# Create your models here.
+class Thread(models.Model):
+    """
+    the model for a thread/conversation
+    """
+    title = models.TextField()
+    date_created = models.DateTimeField()
+    author = models.OneToOneField(User, models.CASCADE)
```

Comment:
This should probably be a `ForeignKey` - one-to-one means each user can have one and only one thread, which is probably not what we want.

Additionally, the use of `CASCADE` on delete is a design decision - as is, if a user is deleted, all of that user's threads will be deleted. Is that what we want?